### ISSUE TITLE
EthPM + Truffle documentation needs to be updated for EthPM v2

### ISSUE THREAD

**Issue by onggunhao** [trufflesuite/trufflesuite.com#398]
[Package Management via EthPM](https://github.com/trufflesuite/trufflesuite.com/blob/master/src/docs/truffle/getting-started/package-management-via-ethpm.md) gives a 404 error for the EthPM registry. The package to be installed also returns a 404. 

It seems that there is a new EthPM v2 and the existing instructions are outdated.

I have pinged @Nick in the `ethpm/lobby` gitter and can work on an update to the guidebook. It seems that EthPM v2 doesn't have a centralized repository, but instead recommends everyone deploy their own authorized repository instead.

I'm part of the Consensys Ethereum Bootcamp, happy to work on editing the docs if someone can give me guidance

Edit: cc @gnidan 

- [ ] Understand changes in EthPM v2
- [ ] Update TruffleSuite instructions

![image](https://user-images.githubusercontent.com/518024/58025681-4396d980-7b15-11e9-95ef-829f760f6fc9.png)

**Comment by gnidan** [comment]
Thanks for raising this @onggunhao! Just to note: Truffle does not currently support EthPM v2. The integration work to make this happen is currently waiting on trufflesuite/truffle#1718, since that effort seeks to make this kind of integration a lot easier.

PRs welcome if anyone wants to make Truffle work with EthPM v2 (prior to the above). We're just looking to apply our team's time efficiently and make sure we're not duplicating work.

**Comment by onggunhao** [comment]
Thanks @gnidan! I'll close this issue since it's already tracked
